codec-http2/src/main/java/io/netty/handler/codec/http2/DefaultHttp2ConnectionDecoder.java

```diff
@@ -282,6 +282,12 @@ public void onHeadersRead(ChannelHandlerContext ctx, int streamId, Http2Headers
                 return;
             }
 
+            if (stream.isTrailersReceived() || stream.isHeadersReceived() && !endOfStream) {
```

Comment:
I don't think this is an issue ... `shouldIgnoreHeadersOrDataFrame` won't let a `null` stream slip by here.